src/js/webrequest.js

```diff
@@ -118,6 +118,12 @@ function onBeforeRequest(details){
  * @returns {*} modified headers
  */
 function onBeforeSendHeaders(details) {
+  if (badger) {
+    setTimeout(function() {
+      return badger.heuristicBlocking.heuristicBlockingAccounting(details);
```

Comment:
No need to use `return`, `heuristicBlockingAccounting` doesn't return anything and nobody will get the returned value out of a `setTimeout` anyway.